sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/DoNotHardcodeCredentials_DefaultValues.cs

```diff
@@ -11,7 +11,7 @@ class Program
 
         private const string secret = "constantValue";
 
-        public void Test()
+        public void Test(string user)
         {
             string password = @"foo"; // Noncompliant {{"password" detected here, make sure this is not a hard-coded credential.}}
```

Comment:
do we have tests with 
```
var something = @"
foo
bar
password
";
```

or combinations with 

`var something = (a = "foo") + (b = "bar");`

and other weird cases like this?